episodes/introduction.md

```diff
@@ -1,28 +1,213 @@
 ---
-title: "Introduction to Open Data"
-teaching: 10
-exercises: 2
+title: "Introduction"
+teaching: 30
+exercises: 6
 ---
 
 :::::::::::::::::::::::::::::::::::::: questions
 
-- What do I need to think about when I'm choosing a dataset to work with?
+* where do I start? /how do i know where to begin?
+* is it normal to feel like my project has gone nowhere and I should start over?
+* my coursework makes sense. what makes research so much harder?
+* this sucks. how can I make sure no one else has to suffer this misery?
+* what can I do to feel like part of a research community?
+* what can I do to help my work have lasting impact?
+* what do I need to think about so that my project doesn't fall apart halfway to the finish line?
 
 ::::::::::::::::::::::::::::::::::::::::::::::::
 
 ::::::::::::::::::::::::::::::::::::: objectives
 
-- Explain the tradeoffs between open and commercial data
-- Describe high-level data needs for a research question
-- Describe the impact of the data production process for XYZ dataset on downstream transformations?
+* describe (/explain?) the phases of a data processing project (/research project?)
+* discuss the benefits of open data principles
+* list(/summarize? briefly describe?) the modules covered in this course
+* identify key considerations in selecting a dataset for an open data project
```

Comment:
these may need weeding? but idk it feels like we really do want folks to come away with all 4 things